Gems/EMotionFX/Code/Source/Editor/PropertyWidgets/ActorJointHandler.cpp

```diff
@@ -225,7 +225,7 @@ namespace EMotionFX
 
         connect(picker, &ActorJointPicker::SelectionChanged, this, [picker]()
         {
-            EBUS_EVENT(AzToolsFramework::PropertyEditorGUIMessages::Bus, RequestWrite, picker);
+            AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast(&AzToolsFramework::PropertyEditorGUIMessages::Bus::Events::RequestWrite, picker);
             AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast(&AzToolsFramework::PropertyEditorGUIMessages::Bus::Handler::OnEditingFinished, picker);
```

Comment:
These EBus calls can be grouped together to form a single call.
```suggestion
            AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast([](AzToolsFramework::PropertyEditorGUIMessages* handler) {
            handler->RequestWrite(picker);
            handler->OnEditingFinished(picker);
            });
```